### PR TITLE
Start cleanup of infra README and add TODOs.

### PR DESCRIPTION
**Pull request by michaelwsherman**

### CODE REVIEW

File: infra/README.md [hunk 1]
```diff
@@ -1,43 +1,54 @@
...
-5. Continue and complete
+1. Go to https://console.firebase.google.com/.
+2. Select "Create a project" and enter the name of your Google Cloud Platform project, then click "Continue". 
+3. If you're using Firebase for the first time, you'll have to add Firebase to one of yor existing Google Cloud projects and confirm the Firebase billing plan.
+4. When prompted to set up Google Analytics respond as you'd like.
+5. Continue and complete.
 
 ### Enable Vertex AI Agent Builder
-Required before start using App Builder services
+Required before starting using App Builder services.
```
Comment (ghchinoy) [inline infra/README.md:22]: Agent Builder

File: infra/README.md [hunk 2]
```diff
@@ -47,55 +58,83 @@ terraform init
...
-4. Set the name for the project and support email for project
-5. Save
+## Step 3 - Setup Google Drive
+(TODO: explain this better)
 
-## Create Gdrive assets
+### Create Folder and Upload Files
+Execute the following script from the `infra` subfolder:
 
-### Create Gdrive assets
```
Comment (ghchinoy) [inline infra/README.md:67]: Google Drive

**Review by ghchinoy — COMMENTED** [review]
Minor product name corrections

File: infra/README.md (hunk 1 above)
Comment (michaelwsherman) [reply infra/README.md]: Good catch, fixed!

File: infra/README.md (hunk 2 above)
Comment (michaelwsherman) [reply infra/README.md:67]: This is a deleted line, and it's fixed elsewhere in the README.

**Pull request merged**